api/src/modules/scenario-interventions/scenario-interventions.service.ts

```diff
@@ -443,7 +451,53 @@ export class ScenarioInterventionsService extends AppBaseService<
     scenarioIntervention.newLocationType = dto.newLocationType;
     scenarioIntervention.newLocationCountryInput = dto.newLocationCountryInput;
     scenarioIntervention.newLocationAddressInput = dto.newLocationAddressInput;
+    scenarioIntervention.createDto = initialDto as unknown as JSON;
```

Comment:
Could this type casting be avoidable typing the property defined in the entity as CreatedInterventionDto?

Reply:
@alexeh at first i tried doing it like that, but that would cause circular dependency and bunch of errors.